docker/docker-compose.yml

```diff
@@ -0,0 +1,50 @@
+version: '3'
+
+volumes:
+  mysql_data:
+      driver: local
+  realm-export.json:
+
+services:
+  mysql:
+      image: mysql:5.7
```

Comment:
Are you doing this for persistence? I can run keycloak (quay.io/keycloak/keycloak , which is the recommended one: https://www.keycloak.org/getting-started/getting-started-docker) without it

Reply:
But then you face a problem that you have to restart the container
`keycloak_1  | Added 'admin' to '/opt/jboss/keycloak/standalone/configuration/keycloak-add-user.json', restart server to load user
keycloak_1  | -b 0.0.0.0`

Reply:
Check, its just nice to have it a bit lighter.

Reply:
sure thing, it should be addressed now, i created a separate realm for rdpgw